src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &.stars--1 .stars__star:nth-of-type(-n + 1),
+  &.stars--2 .stars__star:nth-of-type(-n + 2),
+  &.stars--3 .stars__star:nth-of-type(-n + 3),
+  &.stars--4 .stars__star:nth-of-type(-n + 4),
+  &.stars--5 .stars__star:nth-of-type(-n + 5) {
```

Comment:
Consider using a loop for generating the modifiers for the stars. This will make the code shorter and more maintainable. For example, you can use a `@for` loop in SCSS to iterate from 1 through 5 and generate the `&.stars--#{$i}` classes dynamically.